aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -318,19 +319,34 @@ Tensor imag(const Tensor& self) {
   }
 }
 
-Tensor& conj_out(const Tensor& self, Tensor& result) {
-  return unary_op_impl_out(result, self, conj_stub);
+Tensor resolve_conj(const Tensor& self) {
+  if (!self.is_conj()) { return self; }
+  auto result = at::empty_like(self, self.options());
+  result.set_conj(false);
```

Comment:
LOL, I wonder if I wrote this. I really really hope the `set_conj` here is not actually necessary (and we can replace it with an assert)

Reply:
removed it now that `empty_like` doesn't propagate the conj bit.